apps/meteor/tests/end-to-end/api/livechat/20-messages.ts

```diff
@@ -51,5 +52,17 @@ describe('LIVECHAT - messages', () => {
 				expect(quotedMessageAttachments).to.have.property('text').that.is.equal(agentMsgSentence);
 			}
 		});
+
+		it('it should verify if visitor is receiving a message with a image attachment', async () => {
+			const {
+				room: { _id: roomId },
+				visitor: { token },
+			} = await startANewLivechatRoomAndTakeIt();
+
+			const imgMessage = await uploadFile(roomId, token);
+
+			expect(imgMessage).to.have.property('files');
+			expect(imgMessage.files).to.be.an('array');
```

Comment:
You can combine these 2 expects in one too.

But also would be cool if you checked for the array to contain what u want too